subsys/mpsl/fem/mpsl_fem.c

```diff
@@ -311,41 +311,7 @@ static int fem_nrf21540_gpio_configure(void)
 #endif
 
 #if DT_NODE_HAS_STATUS(MPSL_FEM_SPI_IF, okay)
```

Comment:
This `#if` block can be removed altogether.

Reply:
My reasoning behind leaving it was that when the support for spi control comes, something might need to be done about the spi interface, which would have it's place there. If it's unnecessary and messy, I'll remove the entire block

Reply:
When the day comes (provided that it actually does come) and this line is needed, we'll add it again. No need to keep it now :)

Reply:
Alright, removed the entire block :)